src/main/scala/Garage/Garage.scala

```diff
@@ -1,39 +1,60 @@
 package Garage
 
+import com.mongodb.BasicDBObject
+import org.mongodb.scala.bson.BsonValue
+
 import scala.collection.mutable.ListBuffer
 
 class Garage {
 
-  var vehicles = new ListBuffer[Vehicle]()
-  var employees = new ListBuffer[Employee]()
+  val DB = new DBMethods
+  val connection = new DBConnection
+  val mongoClient = connection.getClient("mongodb://localhost")
```

Comment:
"mongodb://localhost" literal values like this should come from a constants file